test/xUnit/xUnit.tests.csproj

```diff
@@ -23,9 +23,12 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit" Version="2.5.0" />
+    <PackageReference Include="xunit" Version="2.5.3" />
```

Comment:
Moving to 2.5.3 causes xUnit test to fail. I didn't look closely and thus not sure why.